uui/components/filters/PresetPanel/PresetInput.module.scss

```diff
@@ -20,21 +20,34 @@
 
         :global(.uui-icon-accept) {
             &:global(.-clickable) {
-                fill: var(--white);
                 background-color: var(--uui-accent-50);
+                border-radius: var(--uui-border-radius);
+                fill: var(--white);
 
                 &:hover {
                     fill: var(--white);
                     background-color: var(--uui-accent-60);
                 }
+
+                &:active {
+                    fill: var(--white);
+                    background-color: var(--uui-accent-70);
+                }
             }
         }
 
         :global(.uui-icon-cancel) {
-            background-color: var(--uui-secondary-10);
+            background-color: var(--uui-surface-main);
```

Comment:
May be better use here neutral palette instead of surface? And if we supposed that secondary color might not be gray, it's better to use neutral also in cases where you use --uui-secondary

Reply:
Better to change IconContainers to Button